### PR TITLE
support multi_match options

### PR DESCRIPTION
**Pull request by matsu911**
pass through options to multi_match such as type and operator

### CODE REVIEW

File: lib/tirexs/query.ex [hunk 1]
```diff
@@ -43,8 +43,8 @@ defmodule Tirexs.Query do
 
   @doc false
   def multi_match(options) do
-    [query, fields, _] = extract_options(options)
-    [multi_match: [query: query, fields: fields]]
+    [query, fields, options] = extract_options(options)
```
Comment (OpakAlex) [inline lib/tirexs/query.ex]: nice fix ;)

**Comment by stefanahman** [comment]
Fixes https://github.com/Zatvobor/tirexs/issues/261

Ready for merge?

**Comment by OpakAlex** [comment]
@Zatvobor can you push new version?

**Comment by OpakAlex** [comment]
@stefanahman merged. waiting for bump new version.

**Comment by Zatvobor** [comment]
going to release these changes by tomorrow

**Comment by Zatvobor** [comment]
has been released, https://github.com/Zatvobor/tirexs/releases/tag/v0.8.14

**Pull request merged**